packages/syft/src/syft/util/util.py

```diff
@@ -309,7 +310,11 @@ def print_dynamic_log(
     return (finish, success)
 
 
-def find_available_port(host: str, port: int, search: bool = False) -> int:
+def find_available_port(
```

Comment:
Host here is always "localhost", so just ask kernel to give an available port?

`packages/syft/tests/utils/random_port.py`
```
def get_random_port():
    soc = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
    soc.bind(("", 0))
    return soc.getsockname()[1]
```

Reply:
That makes sense. Moving this function to `syft.util.util` for increasing discoverability, and using it instead 🚀